rct229/rulesets/ashrae9012019/ruleset_functions/get_zone_BPF_BAT.py

```diff
@@ -0,0 +1,148 @@
+    LIGHTING_SPACE.WORKSHOP: "UNDETERMINED",
+    LIGHTING_SPACE.ASSISTED_LIVING_FACILITY_CHAPEL: "UNDETERMINED",
+    LIGHTING_SPACE.ASSISTED_LIVING_FACILITY_RECREATION_ROOM_COMMON_LIVING_ROOM: "UNDETERMINED",
+    LIGHTING_SPACE.CONVENTION_CENTER_EXHIBIT_SPACE: "ALL_OTHER",
+    LIGHTING_SPACE.DORMITORY_LIVING_QUARTERS: "MULTIFAMILY",
+    LIGHTING_SPACE.FIRE_STATION_SLEEPING_QUARTERS: "UNDETERMINED",
+    LIGHTING_SPACE.GYMNASIUM_FITNESS_CENTER_EXERCISE_AREA: "UNDETERMINED",
+    LIGHTING_SPACE.GYMNASIUM_FITNESS_CENTER_PLAYING_AREA: "UNDETERMINED",
+    LIGHTING_SPACE.HEALTHCARE_FACILITY_EMERGENCY_ROOM: "HEALTHCARE_HOSPITAL",
+    LIGHTING_SPACE.HEALTHCARE_FACILITY_EXAM_TREATMENT_ROOM: "HEALTHCARE_HOSPITAL",
+    LIGHTING_SPACE.HEALTHCARE_FACILITY_MEDICAL_SUPPLY_ROOM: "HEALTHCARE_HOSPITAL",
+    LIGHTING_SPACE.HEALTHCARE_FACILITY_NURSERY: "HEALTHCARE_HOSPITAL",
+    LIGHTING_SPACE.HEALTHCARE_FACILITY_NURSES_STATION: "HEALTHCARE_HOSPITAL",
+    LIGHTING_SPACE.HEALTHCARE_FACILITY_OPERATING_ROOM: "HEALTHCARE_HOSPITAL",
+    LIGHTING_SPACE.HEALTHCARE_FACILITY_PATIENT_ROOM: "HEALTHCARE_HOSPITAL",
+    LIGHTING_SPACE.HEALTHCARE_FACILITY_PHYSICAL_THERAPY_ROOM: "HEALTHCARE_HOSPITAL",
+    LIGHTING_SPACE.HEALTHCARE_FACILITY_RECOVERY_ROOM: "HEALTHCARE_HOSPITAL",
+    LIGHTING_SPACE.LIBRARY_READING_AREA: "UNDETERMINED",
+    LIGHTING_SPACE.LIBRARY_STACKS: "UNDETERMINED",
+    LIGHTING_SPACE.MANUFACTURING_FACILITY_DETAILED_MANUFACTURING_AREA: "ALL_OTHER",
+    LIGHTING_SPACE.MANUFACTURING_FACILITY_EQUIPMENTROOM: "ALL_OTHER",
+    LIGHTING_SPACE.MANUFACTURING_FACILITY_EXTRA_HIGH_BAY_AREA: "ALL_OTHER",
+    LIGHTING_SPACE.MANUFACTURING_FACILITY_HIGH_BAY_AREA: "ALL_OTHER",
+    LIGHTING_SPACE.MANUFACTURING_FACILITY_LOW_BAY_AREA: "ALL_OTHER",
+    LIGHTING_SPACE.MUSEUM_GENERAL_EXHIBITION_AREA: "ALL_OTHER",
+    LIGHTING_SPACE.MUSEUM_RESTORATION_ROOM: "ALL_OTHER",
+    LIGHTING_SPACE.POST_OFFICE_SORTING_AREA: "ALL_OTHER",
+    LIGHTING_SPACE.RELIGIOUS_FACILITY_FELLOWSHIP_HALL: "ALL_OTHER",
+    LIGHTING_SPACE.RELIGIOUS_FACILITY_WORSHIP_PULPIT_CHOIR_AREA: "ALL_OTHER",
+    LIGHTING_SPACE.RETAIL_FACILITIES_DRESSING_FITTING_ROOM: "RETAIL",
+    LIGHTING_SPACE.RETAIL_FACILITIES_MALL_CONCOURSE: "RETAIL",
+    LIGHTING_SPACE.SPORTS_ARENA_PLAYING_AREA_CLASS_I_FACILITY: "ALL_OTHER",
+    LIGHTING_SPACE.SPORTS_ARENA_PLAYING_AREA_CLASS_II_FACILITY: "ALL_OTHER",
+    LIGHTING_SPACE.SPORTS_ARENA_PLAYING_AREA_CLASS_III_FACILITY: "ALL_OTHER",
+    LIGHTING_SPACE.SPORTS_ARENA_PLAYING_AREA_CLASS_IV_FACILITY: "UNDETERMINED",
+    LIGHTING_SPACE.TRANSPORTATION_FACILITY_BAGGAGE_CAROUSEL_AREA: "ALL_OTHER",
+    LIGHTING_SPACE.TRANSPORTATION_FACILITY_AIRPORT_CONCOURSE: "ALL_OTHER",
+    LIGHTING_SPACE.TRANSPORTATION_FACILITY_TICKET_COUNTER: "ALL_OTHER",
+    LIGHTING_SPACE.WAREHOUSE_STORAGE_AREA_MEDIUM_TO_BULKY_PALLETIZED_ITEMS: "WAREHOUSE",
+    LIGHTING_SPACE.WAREHOUSE_STORAGE_AREA_SMALLER_HAND_CARRIED_ITEMS: "WAREHOUSE",
+}
```

Comment:
See the example lighting to hvac map. To be consistent, we can move this map to a json file.

Reply:
Good point. Addressed